app/src/main/java/de/westnordost/streetcomplete/quests/LeaveNoteDialog.java

```diff
@@ -59,6 +61,20 @@ public void onClick(View v)
 		return view;
 	}
 
+	@Override public void onViewCreated(View view, @Nullable Bundle savedInstanceState)
+	{
+		super.onViewCreated(view, savedInstanceState);
+		if(savedInstanceState == null)
+		{
+			getChildFragmentManager().beginTransaction().add(R.id.attachPhotoFragment, new AttachPhotoFragment()).commit();
+		}
+	}
+
+	private AttachPhotoFragment getAttachPhotoFragment()
+	{
+		return (AttachPhotoFragment) getChildFragmentManager().findFragmentById(R.id.attachPhotoFragment);
```

Comment:
Also, I should probably add nullchecks here. Since adding a child fragment is asynchronous, places where that child fragment is used might be called before the fragment is finally added.